### PR TITLE
add file storage telemetry

### PR DESCRIPTION
**Pull request by neallred**
#### Summary
https://mattermost.atlassian.net/browse/MM-44857

#### Ticket Link
Add file storage to daily activity telemetry event if instance is cloud.

#### Release Note
```release-note
Cloud instances send file storage usage in daily activity telemetry event.
```

### CODE REVIEW

File: services/telemetry/telemetry.go [hunk 1]
```diff
@@ -331,6 +332,14 @@ func (ts *TelemetryService) trackActivity() {
 		activeUsersMonthlyCount = r.Data.(int64)
 	}
 
+	outgoingWebhooksCount, _ = ts.dbStore.Webhook().AnalyticsOutgoingCount("")
```
Comment (marianunez) [inline services/telemetry/telemetry.go:335]: Is this repeated?

File: services/telemetry/telemetry.go [hunk 2]
```diff
@@ -331,6 +332,14 @@ func (ts *TelemetryService) trackActivity() {
...
+
+	if license := ts.srv.License(); license != nil && license.Features.Cloud != nil && *license.Features.Cloud {
+		if usage, err := ts.dbStore.FileInfo().GetStorageUsage(true, false); err == nil {
+			storageBytes = utils.RoundOffToZeroes(float64(usage))
+		}
+	}
```
Comment (marianunez) [inline services/telemetry/telemetry.go:341]: Can we extend the tests at `telemetry_test.go` to include these cloud/non-cloud cases?

File: services/telemetry/telemetry.go [hunk 3]
```diff
@@ -886,21 +898,26 @@ func (ts *TelemetryService) trackPlugins() {
...
+		totalCoreDisabledCount = -1 // -1 to indicate disabled or error
+		totalDisabledCount = -1     // -1 to indicate disabled or error
 	}
 
 	ts.SendTelemetry(TrackPlugins, map[string]interface{}{
 		"enabled_plugins":               totalEnabledCount,
 		"enabled_webapp_plugins":        webappEnabledCount,
 		"enabled_backend_plugins":       backendEnabledCount,
 		"disabled_plugins":              totalDisabledCount,
+		"disabled_default_plugins":      totalCoreDisabledCount,
```
Comment (neallred) [inline services/telemetry/telemetry.go:920]: @sri-byte added `disabled_default_plugins` to this PR per your [request](https://community.mattermost.com/private-core/pl/1h11gtcmqfrnpe3cmyuytgknfr)

**Comment by neallred** [comment]
@marianunez added the requested tests and also add the analytics team's request for disabled core plugins.

File: services/telemetry/telemetry.go [hunk 4]
```diff
@@ -886,21 +897,26 @@ func (ts *TelemetryService) trackPlugins() {
 				if plugin.Manifest.HasWebapp() {
 					webappDisabledCount += 1
 				}
+				if _, isCorePlugin := model.InstalledIntegrationsIgnoredPlugins[plugin.Manifest.Id]; isCorePlugin {
```
Comment (marianunez) [inline services/telemetry/telemetry.go:900]: Interesting, TIL you can disable the core plugins 🤔

**Review by marianunez — APPROVED** [review]
Thanks @neallred! LGTM

File: services/telemetry/telemetry.go (hunk 3 above)
Comment (sri-byte) [reply services/telemetry/telemetry.go:919]: Thank you @neallred this is great!

**Pull request merged**